project-base/tests/FrontendApiBundle/Functional/Login/LoginTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Login;
+
+use Tests\FrontendApiBundle\Test\GraphQlTestCase;
+
+class LoginTest extends GraphQlTestCase
+{
+    public function testPaymentNameByUuid(): void
```

Comment:
I believe the name of the test should be different ;)

Reply:
96da1cf35ecd5d514da9ad2e7f5455b8fc013a8e